reference/content-types/smart_content.rst

```diff
@@ -68,10 +68,6 @@ Parameters
     * - category_root
       - string
       - Root category (key) to display category-tree.
-    * - display_options
```

Comment:
/cc @chirimoya @danrot is this something we forget in 2.0 or was it also not possible in 1.6?

Reply:
TBH I don't know, would have to test it myself...